dockstore-webservice/src/main/java/io/dockstore/webservice/DockstoreWebserviceConfiguration.java

```diff
@@ -96,6 +96,18 @@
     @NotEmpty
     private String gitlabClientSecret;
 
+    @NotEmpty
+    private String zenodoClientID;
+
+    @NotEmpty
+    private String zenodoRedirectURI;
+
+    @NotEmpty
+    private String zenodoUrl;
```

Comment:
This could use some documentation. 
Arguably, the whole class has this issue, but the `zenodoUrl` is starting a new pattern

Reply:
Did you mean documentation in the Dockstore Primer or in dockstore-webservice/src/main/java/io/dockstore/webservice/DockstoreWebserviceConfiguration.java or both?

Reply:
At the time I meant Javadoc, I don't have a strong opinion on the primer.

Reply:
Added documentation in the Dockstore Primer for Zenodo linking